app/src/ui/lib/update-branch.ts

```diff
@@ -62,7 +72,7 @@ export async function updateRebasePreview(
 
   // if we are unable to find any commits to rebase, indicate that we're
   // unable to proceed with the rebase
-  if (commits === null) {
+  if (commitsFromOtherBranch === null) {
```

Comment:
Shouldn't this check for both kinds of commits?
```suggestion
  if (commits === null && commitsFromOtherBranch === null) {
```